mod/metric_answer_lookup/lib/answer/answer_class_methods.rb

```diff
@@ -124,5 +107,28 @@ def unknown? val
     def find_by_answer_id answer_id
       answer_id ? Answer.where(answer_id: answer_id).take : nil
     end
+
+    # convert value format to lookup-table-suitable value
+    # @return nil or String
+    def value_to_lookup value
+      return nil unless value.present?
+      value.is_a?(Array) ? value.join(VALUE_JOINT) : value.to_s
+    end
+
+    # convert value from lookup table to
+    def value_from_lookup string, type
+      type == :multi_category ? string.split(VALUE_JOINT) : string
+    end
+
```

Comment:
perhaps changing `value_card.value` to `value_card.lookup_value` would make this even clearer.